### PR TITLE
Remove deprecated go package

### PR DESCRIPTION
**Pull request by gab-arrobo**

### CODE REVIEW

**Comment by omecproject** [comment]
Can one of the admins verify this patch?

**Comment by thakurajayL** [comment]
retest this please

File: pfcpiface/web_service.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ func (c *ConfigHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	case "PUT":
 		fallthrough
 	case "POST":
-		body, err := ioutil.ReadAll(r.Body)
+		body, err := io.ReadAll(r.Body)
```
Comment (thakurajayL) [inline pfcpiface/web_service.go:53]: Would like to understand reason behind this changes

Comment (gab-arrobo) [reply pfcpiface/web_service.go:53]: The ioutil package was deprecated. Here some text from the ioutil package (https://pkg.go.dev/io/ioutil):
```
Package ioutil implements some I/O utility functions.

Deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.
```

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**